src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -73,12 +68,19 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
   }
 
   const postMaker = postNotifItem?.rawJson?.postMaker;
-  const isOwnPost = postNotifItem?.rawJson?.isOwnPost;
   const isOwnSignedPost = item?.rawJson?.isOwnSignedPost;
 
   const {determinePostType} = useChannelHook();
   const type = determinePostType(postNotifItem);
 
+  const getIsMe = () => {
+    if (item?.channelType === 'ANON_POST_NOTIFICATION')
+      return postNotifItem?.rawJson?.isOwnAnonymousPost;
+    if (item?.channelType === 'POST_NOTIFICATION') return postNotifItem?.rawJson?.isOwnSignedPost;
+
+    return false;
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new function `getIsMe` seems to be replacing the old `isOwnPost` variable. However, it's not clear how `isOwnAnonymousPost` and `isOwnSignedPost` are being set. If these values are not properly initialized, the function might return incorrect results. Please ensure these variables are correctly set before they are used.
```diff
-  const isOwnPost = postNotifItem?.rawJson?.isOwnPost;
+  const getIsMe = () => {
+    if (item?.channelType === 'ANON_POST_NOTIFICATION')
+      return postNotifItem?.rawJson?.isOwnAnonymousPost;
+    if (item?.channelType === 'POST_NOTIFICATION') return postNotifItem?.rawJson?.isOwnSignedPost;
+
+    return false;
+  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->